.pipelines/templates/release-MakeBlobPublic.yml

```diff
@@ -1,80 +1,98 @@
     value: false
-
   steps:
-  - checkout: self
-    clean: true
-    env:
-      ob_restore_phase: true # This ensures checkout is done at the beginning of the restore phase
-
-  - template: /.pipelines/templates/SetVersionVariables.yml@self
-    parameters:
-      ReleaseTagVar: $(ReleaseTagVar)
-      CreateJson: yes
-      UseJson: no
-
-  - pwsh: |
-      Get-ChildItem Env: | Out-String -width 9999 -Stream | write-Verbose -Verbose
-    displayName: 'Capture Environment Variables'
-
-  - pwsh: |
-      $azureRmModule = Get-InstalledModule AzureRM -ErrorAction SilentlyContinue -Verbose
-      if ($azureRmModule) {
-        Write-Host 'AzureRM module exists. Removing it'
-        Uninstall-AzureRm
-        Write-Host 'AzureRM module removed'
-      }
-
-      Install-Module -Name Az.Storage -Force -AllowClobber -Scope CurrentUser -Verbose
-    displayName: Remove AzRM modules
-
-  - task: AzureCLI@2
-    displayName: 'Set blob permissions'
-    inputs:
-      azureSubscription: az-blob-cicd-infra
-      scriptType: 'pscore'
-      scriptLocation: 'inlineScript'
-      inlineScript: |
-        az storage container set-permission --account-name $(StorageAccount) --name $(azureVersion) --public-access blob
-        az storage container set-permission --account-name $(StorageAccount) --name $(azureVersion)-gc --public-access blob
+    - checkout: self
+      clean: true
+      env:
+        ob_restore_phase: true # This ensures checkout is done at the beginning of the restore phase
+
+    - template: /.pipelines/templates/SetVersionVariables.yml@self
+      parameters:
+        ReleaseTagVar: $(ReleaseTagVar)
+        CreateJson: yes
+        UseJson: no
+
+    - pwsh: |
+        Get-ChildItem Env:
+      displayName: 'Capture Environment Variables'
+
+    - pwsh: |
+        $azureRmModule = Get-InstalledModule AzureRM -ErrorAction SilentlyContinue -Verbose
+        if ($azureRmModule) {
+          Write-Host 'AzureRM module exists. Removing it'
+          Uninstall-AzureRm
+          Write-Host 'AzureRM module removed'
+        }
+
+        Install-Module -Name Az.Storage -Force -AllowClobber -Scope CurrentUser -Verbose
+      displayName: Remove AzRM modules
+
+    - task: AzurePowerShell@5
+      displayName: Copy blobs to PSInfra storage
+      inputs:
+        azureSubscription: az-blob-cicd-infra
+        scriptType: inlineScript
+        azurePowerShellVersion: LatestVersion
+        pwsh: true
+        inline: |
+          $sourceStorageAccountName = '$(StorageAccount)'
+          $destinationStorageAccountName = '$(PSInfraStorageAccount)'
+          $destinationContainerName = '$web'
+          $destinationPrefix = '$(ReleaseTagVar)'
+
+          $sourceContext = New-AzStorageContext -StorageAccountName $sourceStorageAccountName
+          Write-Verbose -Verbose "Source context: $($sourceContext.BlobEndPoint)"
+
+          $destinationContext = New-AzStorageContext -StorageAccountName $destinationStorageAccountName
```

Comment:
Consider checking if destination container exists and possibly delete it before copying into it. So, on re-runs we are sure that everything is new.